static/scss/answers/cards/product-prominentimage.scss

```diff
@@ -55,14 +55,16 @@
   {
     display: flex;
     align-items: flex-start;
+    justify-content: center;
     width: 100%;
     flex-shrink: 0;
   }
 
   &-img
   {
     width: 100%;
-    height: auto;
+    height: 300px;
+    object-fit: cover;
```

Comment:
does this work on ie11? I had issues getting object-fit to work on it before

Reply:
you are right, it didn't work. I updated to use a div with background image instead of img tag, so we can use background-position and background-size similiar to object-position and object-fit. this should work in all browser